src/effects/builtin/flangereffect.cpp

```diff
@@ -5,10 +5,6 @@
 #include "util/math.h"
 
 namespace{
-
-// Gain correction was verified with replay gain and default parameters
-const double kGainCorrection = 1.4125375446227544; // 3 dB
```

Comment:
Are you sure that removing this is correct? I have the impression that it's now much louder when the flanger is active.